tests/test_querybuilder.py

```diff
@@ -66,6 +66,13 @@ def test_querybuilder_core():
     assert q.get_sql(PARAM_PLPY) == exp
 
 
+def test_querybuilder_inline():
+    args = {'list': [1, 2], 'dict': {'a': 1}, 'none': None}
```

Comment:
test should contain float as well, as this is also covered by your logic. And something "different" for final return.

Reply:
Thanks, good points.

Did a research on float issue, whether it still needed, it turns out its py2-only, fixed in 3.2 - https://bugs.python.org/issue9337

So it seems there is no need to handle float specially in py3.

Reply:
Yeah, my point is just add all possible value types into test.